src/features/Chat/components/ActiveWindow/Header/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+
+import type { ChatBuddy } from '@/features/Chat/chatSlice';
+
+import styled from 'styled-components';
+import ArchivedIcon from '@/static/icons/archived-chats.svg';
+import Buttons from './Buttons';
+import BlockedIcon from '@/static/icons/blocked-chats.svg';
+import { palette } from '@/components/variables';
+import { Profile as ProfileIcon } from '@/components/Icons/Profile';
+import Search from './Search';
+import Text from '@/components/Text';
+
+type Props = {
+  chat: ChatBuddy;
+};
+
+const Header = ({ chat }: Props) => {
+  const [showSearch, setShowSearch] = useState(false);
+  const showSearchBar = () => setShowSearch(true);
+  const hideSearchBar = () => setShowSearch(false);
+
+  const icons = {
+    ok: <ProfileIcon color="purpleDark" />,
+    archived: <img src={ArchivedIcon} />,
+    banned: <img src={BlockedIcon} />,
+  };
+
+  return (
+    <Container>
+      <ProfileInfo>
+        {icons[chat.status]}
+        <MentorName variant="h2">{chat.displayName}</MentorName>
+        <MentorBio variant="p">{chat.status}</MentorBio>
+      </ProfileInfo>
+      {showSearch ? (
```

Comment:
I like that all boolean values are prefixed with "is", or "should".
showSearch sounds like a function.
No need to fix it now, next PR is fine =)

Reply:
Like maybe a better name would be "isSearchShown" and the setter "setIsSearchShown"

Reply:
This is true! Especially when I have a function that is also named "showSearch". I'll change this!